### DISCUSSION TITLE
Priview in Dashboard Media not working

### DISCUSSION THREAD

**Discussion by siddiksuryanaLearning** [awcodes/filament-curator#358] [Q&A]
![Screenshot from 2023-11-29 19-53-36](https://github.com/awcodes/filament-curator/assets/99642687/4b2eb93c-468f-4dcd-b65f-db2549dae3e5)

![Screenshot from 2023-11-29 19-54-43](https://github.com/awcodes/filament-curator/assets/99642687/dc0bf125-bcaa-4d41-9055-a3187790a6b8)

**Comment by awcodes** [comment]
what console errors are you seeing?

**Reply by siddiksuryanaLearning** [reply]
in priview dasboard media, it hit to : http://127.0.0.1:8000/curator/media/df7fecb7-4d48-463c-8cf4-c7a24569ce89.png
but my picture storage in http://127.0.0.1:8000/storage/media/df7fecb7-4d48-463c-8cf4-c7a24569ce89.png
my quesion is, how to replace source files sir ? from http://127.0.0.1:8000/curator/media/ to http://127.0.0.1:8000/storage/media/

**Comment by awcodes** [comment]
The curator/media URL is correct. That is running through Glide to generate the appropriate sizes. It is a proxy URL.

Make sure you have GD on your server / localhost.
